packages/nr-metadata/nr_metadata/ui/semantic-ui/js/nr/search/components/ResultsItemAccessStatus.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Image } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+const iconsObject = {
+  c_abf2: "zamky_open_access.svg",
```

Comment:
Not sure if we can use these assets outside of NR docs @mesemus

Reply:
@mirekys There was a comment to put validation schema back to nr-docs, but now it is gone. I already moved it back, is it OK or not please ?:)

Reply:
Wow you are lightning fast :) sorry about that, original comment was about moving it under [`documents`](https://github.com/Narodni-repozitar/nr-metadata/tree/main/nr_metadata/documents) package, but by further reviewing the code structure, that would mean way to much additional boilerplate needed to be created just for basically this single file, so I took that back, it could be as it was, alteast for now